Adyen/UI/Form/Items/Basic Items/FormContainerItem.swift

```diff
@@ -61,8 +61,8 @@ public class FormContainerItem: FormItem {
 extension FormItem {
 
     /// :nodoc:
-    public func withPadding(padding: UIEdgeInsets) -> FormContainerItem {
-        FormContainerItem(content: self, padding: padding)
+    public func wrapped() -> FormContainerItem {
```

Comment:
I would prefer a different name maybe 🤔
`wrapped` doesn't really explains what it does.

How about `addNaturalMargins` or something?

Reply:
fixed